webrender/src/device.rs

```diff
@@ -46,6 +46,12 @@ pub const TEXTURE_HEIGHT: usize = 8;
 pub const MAX_INSTANCE_COUNT: usize = 1024;
 
 pub type TextureId = u32;
+pub type FBOId = u32;
```

Comment:
any reason we can't keep using the old types here? we need to keep the changes to the minimum